operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,93 @@
+:_content-type: ASSEMBLY
+[id="authentication-provider-structure"]
+= Authentication provider structure
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+An authentication provider connects to a third-party source of user identity, for example, an identity provider or IDP, gets the user identity, issues a token based on that identity, and returns the token to RHACS. 
```

Comment:
```suggestion
An authentication provider connects to a third-party source of user identity, for example, an identity provider (IDP), gets the user identity, issues a token based on that identity, and returns the token to {rh-rhacs-first}. 
```

Reply:
Dictionary: https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki/rh_dictionary_letter_i#:~:text=idP,Identity%20Provider

Reply:
this contradicts what @kcarmichael08 proposed :(